debian/control

```diff
@@ -0,0 +1,14 @@
+Source: sanoid
+Section: unknown
+Priority: optional
+Maintainer: Dustin L. Howett <dustin@howett.net>
+Build-Depends: debhelper (>= 9)
+Standards-Version: 3.9.8
+Homepage: https://github.com/jimsalterjrs/sanoid
+Vcs-Git: https://github.com/jimsalterjrs/sanoid.git
+Vcs-Browser: https://github.com/jimsalterjrs/sanoid
+
+Package: sanoid
+Architecture: all
+Depends: ${misc:Depends}, ${perl:Depends}, zfsutils-linux, libconfig-inifiles-perl
+Description: Policy-driven snapshot management and replication tools
```

Comment:
there's an opportunity for a longer description here, it'll just have to be indented by one space so the control parser doesn't eat it